packages/host/app/services/realm-server.ts

```diff
@@ -110,8 +118,12 @@ export default class RealmServerService extends Service {
     window.localStorage.removeItem(sessionLocalStorageKey);
   }
 
+  @cached
   get availableRealmURLs() {
-    return [...this._userRealmURLs, ...this._catalogRealmURLs];
+    return new TrackedArray([
+      ...this._userRealmURLs,
+      ...this._catalogRealmURLs,
+    ]);
```

Comment:
splatting out the tracked arrays into a regular array was destroying the reactivity for this property

Reply:
Why do we need/want multiple levels of TrackedArray? I think this is something we are trying to avoid.

Reply:
Sure, I’m not too sure why there were 2 separate arrays for this to begin with, but I can refactor that.

Reply:
I refactored so there is only a single tracked array that manages all the available realms. @lukemelia can you re-review?